src/javascript/JContent/ContentRoute/ContentLayout/Upload/UploadItem/Tags.jsx

```diff
@@ -0,0 +1,29 @@
+import {uploadStatuses, TAG_ERROR} from '../Upload.constants';
+import React from 'react';
+import {LocalOffer} from '@material-ui/icons';
+import {Typography} from '@jahia/design-system-kit';
+import {isImageFile} from '../../ContentLayout.utils';
+
+const Tags = props => {
+    const {classes, status, t, file} = props;
+    const tags = props.numTags;
+    let content = null;
+    let tagText = (tags === TAG_ERROR) ? '' : ((tags > 0) ? t('jcontent:label.contentManager.fileUpload.tags') : t('jcontent:label.contentManager.fileUpload.tag'));
+
+    if (window.contextJsParameters.config.links.tagImageOnUpload === 'true' && isImageFile(file.name)) {
```

Comment:
A bit of a code smell to be comparing a boolean as string. Perhaps change the type returned to boolean? See comment below

Reply:
Gotcha will double check that one